dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/transform/ContentletTransformerTest.java

```diff
@@ -235,8 +298,256 @@ public void Test_Constant_fields_are_in_map() throws Exception {
         
     }
 
-    
-    
+
+    @DataProvider
+    public static Object[] listTestCases() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)